suggestedimages/routes.py

```diff
@@ -2,36 +2,50 @@
     Blueprint, flash, redirect, render_template, request, url_for
 )
 import urllib
+from collections import namedtuple
+
 
 from . import search
 from .util import StrInLanguage
 from .locales import Locale
 
+LangOption = namedtuple("LangOption", "value label")
+
 bp = Blueprint('main', __name__)
 
 @bp.route('/', methods=('GET',))
 def index():
     title = request.args.get('title')
-    if not title:
-        return render_template('index.html', results=[], locale=Locale())
-
     wikt = request.args.get('wikt')
     locale = Locale(wikt) if wikt else Locale()
     lang = request.args.get('lang') or locale.language
-    lang_str = StrInLanguage(title, lang=lang)
+
+    language_options = [
+        LangOption(
+            value = lang,
+            label = locale.language_names[lang] + f' [{lang}]'
+        ) for lang in
+            locale.language_names.keys()
+    ]
```

Comment:
I can understand this block of code above so I guess it's good

Reply:
Huh, I'm not sure if this means everything else is bad?